src/app/home/home.component.scss

```diff
@@ -1,29 +1,56 @@
-.container {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-
-    .card {
-        align-items: center;
-        margin: 1rem;
-        width: 500px;
-
-        span {
-            border: 1px solid rgba(0,0,0,.25);
+.template {
+    padding: 1rem;
+    .heading {
+        text-align: center;
+        margin: 1rem 0;
+    }
+    .container {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+        max-width: 100%;
+    
+        .card {
+            align-items: center;
             margin: 1rem;
-            width: 136px;
-            border-radius: 50%;
+            width: 500px;
+            border: 1px solid #00000f;
```

Comment:
Would have been nice to use constants for scss variables.

Reply:
done